Makefile

```diff
@@ -50,16 +64,32 @@ ut-circle:
 	nosetests calico_containers/tests/unit -c nose.cfg \
 	--cover-html-dir=dist --with-xunit --xunit-file=/circle_output/output.xml'
 
-st: binary
-	docker save --output calico_containers/calico-node.tar calico/node
+calico_containers/busybox.tar:
 	docker pull busybox:latest
 	docker save --output calico_containers/busybox.tar busybox:latest
+
+calico_containers/calico-node.tar: caliconode.created
+	docker save --output calico_containers/calico-node.tar calico/node
+
+st: binary calico_containers/busybox.tar calico_containers/calico-node.tar run-etcd
 	dist/calicoctl checksystem --fix
-	nosetests calico_containers/tests/st/ -sv --nologcapture
+	nosetests calico_containers/tests/st/ -sv --nologcapture --with-timer
+
+fast-st: binary calico_containers/busybox.tar calico_containers/calico-node.tar run-etcd
+	nosetests calico_containers/tests/st/ -sv --nologcapture --with-timer -a '!slow'
+
+run-etcd:
+	-docker rm -f calico-etcd
+	docker run --detach \
+	--publish 2379:2379 \
+	--name calico-etcd quay.io/coreos/etcd:v2.0.11 \
+	--advertise-client-urls http://$(IP_ADDRESS):2379 \
```

Comment:
Need a way to override the `IP_ADDRESS` to one of your choosing.  Docker chooses 10.0/16 as a bridge subnet and on Vagrant machines the default NAT interface is 10.0.2.15.  See `get_ip()` in `utils.py`